config/locales/en.yml

```diff
@@ -511,6 +511,12 @@ en:
     new:
       mark_offline: Take instrument offline
 
+  products:
+    user_notes_field_mode:
+      hidden: "No"
```

Comment:
`No` is considered `false` in YAML. So it needs the quotes to make sure it's treated as a string.